### PR TITLE
[FSTORE-528] Update data validation for 3.1

### PR DESCRIPTION
**Pull request by vatj**

### CODE REVIEW

File: integrations/great_expectations/fraud_batch_data_validation.ipynb [hunk 1]
```diff
@@ -623,49 +656,109 @@
...
       "outputs": []
     },
     {
       "cell_type": "markdown",
       "source": [
-        "You can also retrieve validation report summaries via the python client. Feature Groups provide a convenience method to get the latest summary uploaded for quick prototyping."
+        "\n",
+        "You are however more likely to be in one of two use cases: \n",
+        "- Check the latest validation report because something went horribly wrong and you have been tasked with sorting it out.\n",
+        "- Check the history for a particular Expectation to understands exactly when things started to go horribly wrong.\n",
```
Comment (moritzmeister) [inline integrations/great_expectations/fraud_batch_data_validation.ipynb:684]:
```suggestion
        "- Check the history for a particular Expectation to understand exactly when things started to go horribly wrong.\n",
```

File: integrations/great_expectations/fraud_batch_data_validation.ipynb [hunk 2]
```diff
@@ -623,49 +656,109 @@
...
+        "\n",
+        "Luckily Hopsworks provide convenience methods for both cases 🙂"
```
Comment (moritzmeister) [inline integrations/great_expectations/fraud_batch_data_validation.ipynb:686]:
```suggestion
        "Luckily Hopsworks provides convenience methods for both cases 🙂"
```

File: integrations/great_expectations/fraud_batch_data_validation.ipynb [hunk 3]
```diff
@@ -623,49 +656,109 @@
...
+      "source": [
+        "# Create a quick history\n",
+        "from time import sleep\n",
+        "from datetime import datetime\n",
+        "\n",
+        "start_time = datetime.now()\n",
+        "\n",
+        "for i in range(0, 10):\n",
+        "  job, report = trans_fg.insert(trans_df[i*10:(i+1)*10], write_options={\"wait_for_job\": False})\n",
+        "  sleep(1) # Force sleep to avoid uploading report with same timestamp\n",
```
Comment (moritzmeister) [inline integrations/great_expectations/fraud_batch_data_validation.ipynb:716]: It's really that fast?

File: integrations/great_expectations/fraud_batch_data_validation.ipynb [hunk 4]
```diff
@@ -720,15 +813,72 @@
...
+        "                    \"min_value\": -100,\n",
+        "                    \"max_value\": -20\n",
+        "                },\n",
+        "                meta= {\n",
+        "                    \"expectationId\": new_expectation.meta[\"expectationId\"]\n",
+        "                }\n",
+        "            )\n",
+        "        )\n",
+        "\n",
+        "# You can get the expectation from the backend to double check change have indeed been persisted\n",
```
Comment (moritzmeister) [inline integrations/great_expectations/fraud_batch_data_validation.ipynb:859]:
```suggestion
        "# You can get the expectation from the backend to double check that the change has indeed been persisted\n",
```

**Review by moritzmeister — CHANGES_REQUESTED** [review]
Just a few spelling/typos :)

File: integrations/great_expectations/fraud_batch_data_validation.ipynb (hunk 3 above)
Comment (vatj) [reply integrations/great_expectations/fraud_batch_data_validation.ipynb:716]: Ja, there are not that many requests being made and it is a 10 rows DataFrame so...

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**